### ISSUE TITLE
Checks attempted when AAC not active

### ISSUE THREAD

**Issue by SlimeDog** [Photon-GitHub/AACAdditionPro#421]
Spigot 1.16.4
AACAdditionPro 6.5.0
AAC not installed

The four indicated checks are enabled in configuration. We switch are testing between AAC and Spartan, so would like to leave them enabled. However, the checks should not be attempted if AAC is not active.

```
[14:14:21] [Server thread/ERROR]: [AACAdditionPro] Plugin AACAdditionPro v6.5.0 has failed to register events for class de.photon.aacadditionpro.modules.checks.AutoEat because me/konsolas/aac/api/AACCustomFeature does not exist.
[14:14:21] [Server thread/ERROR]: [AACAdditionPro] Plugin AACAdditionPro v6.5.0 has failed to register events for class de.photon.aacadditionpro.modules.checks.AutoPotion because me/konsolas/aac/api/AACCustomFeature does not exist.
[14:14:21] [Server thread/ERROR]: [AACAdditionPro] Plugin AACAdditionPro v6.5.0 has failed to register events for class de.photon.aacadditionpro.modules.checks.scaffold.Scaffold because me/konsolas/aac/api/AACCustomFeature does not exist.
[14:14:21] [Server thread/ERROR]: [AACAdditionPro] Plugin AACAdditionPro v6.5.0 has failed to register events for class de.photon.aacadditionpro.modules.checks.tower.Tower because me/konsolas/aac/api/AACCustomFeature does not exist.
```

**Comment by Photon-GitHub** [comment]
Fixed in 6.5.1

**Comment by SlimeDog** [comment]
Confirmed fixed in 6.5.1 with AAC not active. Thanks for fast work.
